docs/overview.rst

```diff
@@ -0,0 +1,39 @@
+Overview
+========
+
+A command-line tool to help manage EthPM packages and registries.
+
+
+WARNING
+
+``EthPM CLI`` is currently in public Alpha:
+
+- It is expected to have bugs and is not meant to be used in production 
+- Things may be ridiculously slow or not work at all
+
+
+Installing EthPM Packages
+-------------------------
+
+.. argparse::
+   :ref: ethpm_cli.parser.parser
+   :prog: ethpm
+
+
+URIs
+----
+
+ethPM Cli supports the following URI schemes.
+
+- IPFS
+
+  - ``ipfs://[IPFS_HASH]``
+
+- Etherscan
+
+  - ``etherscan://[CONTRACT_ADDRESS]/[CHAIN_ID]``
+  - ``CONTRACT_ADDRESS`` and ``CHAIN_ID`` must represent a `Verified Contract <https://etherscan.io/contractsVerified>`_ on Etherscan.
```

Comment:
I think you should specifically define `CHAIN_ID` and list the id's for common public chains.